src/shared/config-db.ts

```diff
@@ -183,60 +85,153 @@ export const DatabaseConfigScheme = {
   client: {
     doc: 'Which database client should we use',
     format: ['mysql', 'sqlite3'],
-    default: 'sqlite'
+    default: 'sqlite',
+    env: 'DB_CLIENT'
   },
   version: {
     doc: 'What database version should we use',
     format: String,
-    default: '5.5'
+    default: '5.5',
+    env: 'DB_VERSION'
   },
   useNullAsDefault: {
     doc: 'whether or not to use null for everything not specified',
     format: 'Boolean',
-    default: false
+    default: false,
+    env: 'DB_USE_NULL_AS_DEFAULT'
   },
   connection: {
-    doc: 'Connection object specifying properties like host, port, user etc.',
-    format: DbConnectionFormat.name,
-    default: null
+    filename: {
+      doc: 'Database filename',
+      format: '*',
+      default: null,
+      env: 'DB_CONNECTION_FILENAME'
+    },
+    host: {
+      doc: 'Database host',
+      format: '*',
+      default: null,
+      env: 'DB_CONNECTION_HOST'
+    },
+    port: {
+      doc: 'Database port',
+      format: '*',
+      default: null,
+      env: 'DB_CONNECTION_PORT'
+    },
+    user: {
+      doc: 'Database user',
+      format: '*',
+      default: null,
+      env: 'DB_CONNECTION_USER'
+    },
+    password: {
+      doc: 'Database password',
+      format: '*',
+      default: null,
+      env: 'DB_CONNECTION_PASSWORD'
+    },
+    database: {
+      doc: 'Database name',
+      format: '*',
+      default: null,
+      env: 'DB_CONNECTION_NAME'
+    },
+    timezone: {
+      doc: 'Database timezone',
+      format: '*',
```

Comment:
Do they have a format for timezone? or perhaps this should be an enum?

Reply:
No. Since `knex` services connecting to multiplate databases I think the format can be a varied set of strings.
Updated it to be a `String`